### PR TITLE
Fix (src/Forms/FixCommonErrors.cs)

### PR DESCRIPTION
**Pull request by ivandrofly**
# Status
- [x] Ready
- [x] Test

### CODE REVIEW

File: src/Forms/FixCommonErrors.cs [hunk 1]
```diff
@@ -4776,7 +4772,8 @@ private void ToolStripMenuItemDeleteClick(object sender, EventArgs e)
                         int number = Convert.ToInt32(item.SubItems[1].Text);
                         if (number > firstIndex)
                             number -= subtitleListView1.SelectedItems.Count;
-                        deSelectedFixes.Add(number + item.SubItems[2].Text + item.SubItems[3].Text);
```
Comment (ivandrofly) [inline src/Forms/FixCommonErrors.cs]: Note that `number` can be `>` than `firstIndex` but we are not sure if `subtitleListView1.SelectedItems.Count` if going to be `<` than number so  `number -= subtitleListView1.SelectedItems.Count` can be negative

File: src/Forms/FixCommonErrors.cs [hunk 2]
```diff
@@ -4802,10 +4799,13 @@ private void ToolStripMenuItemDeleteClick(object sender, EventArgs e)
                 Next();
 
                 // restore de-selected fixes
-                foreach (ListViewItem item in listViewFixes.Items)
+                if (deSelectedFixes.Count > 0)
```
Comment (ivandrofly) [inline src/Forms/FixCommonErrors.cs]: Can be *0* if `listViewFixes.SelectedItem` was included in deletion, so there is no need to run the *0(n)*

**Comment by niksedk** [comment]
I've also tested some basic stuff (testing is most important:)

**Comment by ivandrofly** [comment]
Indeed :)

**Pull request merged**